lte/gateway/c/oai/include/s1ap_types.h

```diff
@@ -29,6 +29,7 @@ struct enb_description_s;
 
 #define S1AP_TIMER_INACTIVE_ID (-1)
 #define S1AP_UE_CONTEXT_REL_COMP_TIMER 1  // in seconds
+#define S1AP_UE_HANDOVER_NULL_ENB 0
```

Comment:
what is the usage for this? I don't see this being used at the moment, also if this can be updated to reflect the usage (e.g `S1AP_UE_HANDOVER_NULL_ENB_ID`?)

Reply:
Good catch, no longer needed.